contrib/go-pg/pg_go_test.go

```diff
@@ -0,0 +1,49 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package pg
+
+import (
+	"context"
+	"testing"
+
+	"github.com/go-pg/pg/v10"
+	"github.com/stretchr/testify/assert"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/ext"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/mocktracer"
+	"gopkg.in/DataDog/dd-trace-go.v1/ddtrace/tracer"
+)
+
+func GetPostgresConnection() *pg.DB {
```

Comment:
This can be unexported.
```suggestion
func getPostgresConnection() *pg.DB {
```